tests/cuda/test_cca.cpp

```diff
@@ -52,16 +56,21 @@ cca_function_t get_f_with(traccc::clustering_config cfg) {
         copy.setup(cells_buffer)->wait();
         copy(vecmem::get_data(cells), cells_buffer)->wait();
 
-        auto measurements_buffer = cc(cells_buffer, dd_buffer);
+        auto [measurements_buffer, cluster_buffer] =
+            cc(cells_buffer, dd_buffer,
+               traccc::device::clustering_keep_disjoint_set{});
         traccc::measurement_collection_types::host measurements{&host_mr};
         copy(measurements_buffer, measurements)->wait();
 
+        traccc::edm::silicon_cluster_collection::host clusters{host_mr};
+        copy(cluster_buffer, clusters)->wait();
```

Comment:
@krasznaa trying to get your EDM code to work and this is segfaulting; any ideas?